src/components/layouts/Sidebar.tsx

```diff
@@ -3,7 +3,23 @@ import { HashRouter as Router, Link } from 'react-router-dom';
 import './layouts.style.css';
 
 export default class Sidebar extends React.Component<{}> {
-  public render() {
+  state = {
+    showBenchmarkSubmenu: false
+  };
+
```

Comment:
@muskankhedia Follow the structure for the tsx files, as in Benchmarks.tsx file.